### PR TITLE
CDAP-4070 Wait for Kafka server to startup before starting LogSaver tests

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-4070
Build - http://builds.cask.co/browse/CDAP-DUT3128-2

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM :+1:

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/KafkaTestBase.java [hunk 1]
```diff
@@ -94,4 +117,64 @@ private static Properties generateKafkaConfig() throws IOException {
 
     return prop;
   }
+
+  private static void waitForKafkaStartup(String zkConnectString, long timeoutMills) throws Exception {
```
Comment (chtyim) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/KafkaTestBase.java]: There is a easier way to wait for Kafka service to start. You can use `ZKBrokerService` to help:

```java
final BrokerService brokerService = new ZKBrokerService(zkClient);
brokerService.startAndWait();
Tasks.waitFor(true, new Callable<Boolean>() {
  public Boolean call() throws Exception {
    return Iterables.size(brokerService.getBrokers());
  }
}, 10, TimeUnit.SECONDS, 100, TimeUnit.MILLISECONDS);
```

**Comment by poornachandra** [comment]
@chtyim Updated the check with `ZKBrokerService`, can you take another look?

**Comment by chtyim** [comment]
LGTM

**Comment by poornachandra** [comment]
Build is green, merging.

**Pull request merged**